staging/src/k8s.io/client-go/tools/cache/controller.go

```diff
@@ -167,9 +167,9 @@ func (c *controller) processLoop() {
 //  * OnUpdate is called when an object is modified. Note that oldObj is the
 //      last known state of the object-- it is possible that several changes
 //      were combined together, so you can't use this to see every single
-//      change. OnUpdate is also called when a re-list happens, and it will
-//      get called even if nothing changed. This is useful for periodically
-//      evaluating or syncing something.
+//      change. OnUpdate is also called when a re-list and resync happens, and
```

Comment:
what is the difference between re-list and resync ? They are the same as far as i understand ?

Reply:
re-list refetches current items from the apiserver and adds them to the store, resulting in Add events for new items, Delete events for removed items, and Update events for all existing items. 

resync sends Update events for all items currently in the store

Reply:
Great thanks for explaininginfo